tests/conftest.py

```diff
@@ -204,25 +202,22 @@ def _get_contract(w3, source_code, *args, **kwargs):
     value_in_eth = kwargs.pop('value_in_eth', 0)
     value = value_in_eth * 10**18 if value_in_eth else value  # Handle deploying with an eth value.
     gasPrice = kwargs.pop('gasPrice', 0)
-    deploy_transaction = {
+
+    c = w3.eth.contract(abi=abi, bytecode=bytecode)
+    deploy_transaction = c.constructor(*args, **kwargs)
```

Comment:
Should join `kwargs` with `tx_info` instead? Then there would be no reason to do `gasPrice = kwargs.pop('gasPrice', 0)`. There really isn't a point to `kwargs` here I don't think?